core/src/main/java/fr/sncf/osrd/stdcm/graph/STDCMPostProcessing.java

```diff
@@ -47,6 +49,7 @@ STDCMResult makeResult(
         var mergedEnvelopes = STDCMUtils.mergeEnvelopeRanges(ranges);
         var departureTime = computeDepartureTime(ranges, startTime);
         var stops = makeStops(ranges);
+        stops.addAll(makeOpStops(infra, trainPath));
```

Comment:
Is there a possibility of there being duplicates in this list ? When `makeOperationalWaypoints` is called in `PathfindingResultConverter.makePathWaypoint`, the combination of the two lists passes through a call of `sortAndMergeDuplicates`.